src/components/Steps/__tests__/Steps.spec.ts

```diff
@@ -0,0 +1,29 @@
+import '@testing-library/jest-dom';
+import { render, within } from '@testing-library/vue';
+
+import Steps from '../Steps.vue';
+
+type PropsType = InstanceType<typeof Steps>['$props'];
+
+const DEFAULT_PROPS: PropsType = {
+  items: [{ label: 'Step 1' }, { label: 'Step 2' }, { label: 'Step 3' }]
+};
+
+describe('Steps', () => {
+  let props: PropsType;
+
+  beforeEach(() => {
+    props = { ...DEFAULT_PROPS };
+  });
+
+  it('renders', () => {
```

Comment:
does anything happen when clicking on a step? does it get active? does it emit an event?

Reply:
Given that it supports model values, IMO it's probably better to use watchers on the model, anyway, since the only real event emission in this case would be the model changing, right?

Reply:
Interesting thought, I think you're right. Watching it change would probably be the most useful